kernel/futex.c

```diff
@@ -269,6 +280,11 @@ int myst_futex_wake(int* uaddr, int val)
     return ret;
 }
 
+int myst_futex_wake(int* uaddr, int val)
+{
+    return myst_futex_wake_ops(uaddr, val, FUTEX_BITSET_MATCH_ANY);
```

Comment:
Does this function now require checking of futex-bitsets? If so, it modifies its original behavior for the non-bitset case, which could introduce instability. I would prefer that existing code paths remain exactly as before.